src/client/theme/SearchBar/SearchBar.tsx

```diff
@@ -246,12 +260,19 @@ export default function SearchBar({
         onBlur={onInputBlur}
         onChange={onInputChange}
         ref={searchBarRef}
+        value={inputValue}
       />
       <LoadingRing className={styles.searchBarLoadingRing} />
-      <div className={styles.searchHintContainer}>
-        <kbd className={styles.searchHint}>{isMac ? "⌘" : "ctrl"}</kbd>
-        <kbd className={styles.searchHint}>K</kbd>
-      </div>
+      {inputValue !== "" ? (
+        <button className={styles.searchClearButton} onClick={onClearSearch}>
+          ✕
```

Comment:
I re-evaluated the ✕ and it actually is fine so maybe it is best! The native clear button was kind of supported but required a lot of css for it to work in for example firefox so I think maybe this way is still the easiest for us?

Reply:
Yes, I agree.